suite-common/message-releases/releases/test-releases.v1.json

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
This is an example of how the releases could be, it should be pretty much ready, but we still will go throw it to make it contain all the really required information for the releases and not more or less.